Description: Drop edgegrid from requirements
 Its optional so no need to have a hard dependency
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

--- a/requirements.txt
+++ b/requirements.txt
@@ -48,4 +48,3 @@ debtcollector>=1.19.0 # Apache-2.0
 os-win>=4.1.0 # Apache-2.0
 monasca-statsd>=1.4.0 # Apache-2.0
 futurist>=1.2.0 # Apache-2.0
-edgegrid-python>=1.1.1 # Apache-2.0
--- a/designate.egg-info/requires.txt
+++ b/designate.egg-info/requires.txt
@@ -7,7 +7,6 @@ WebOb>=1.7.1
 Werkzeug>=0.9
 debtcollector>=1.19.0
 dnspython>=1.16.0
-edgegrid-python>=1.1.1
 eventlet>=0.26.1
 futurist>=1.2.0
 greenlet>=0.4.15
